src/vs/base/browser/ui/list/listView.ts

```diff
@@ -1544,6 +1545,7 @@ export class ListView<T> implements IListView<T> {
 			this.domNode.parentNode.removeChild(this.domNode);
 		}
 
-		dispose(this.disposables);
+		this.dragOverAnimationDisposable?.dispose();
```

Comment:
You don't need this as it's handed via the new arg as part of `this.disposables.dispose();`

Reply:
This is another one: https://github.com/microsoft/vscode/blob/8162d644e7f01f9defc29ae026b724d7b0067395/src/vs/base/browser/ui/list/listView.ts#L1293-L1294